src/test/scala/edu/ie3/simona/model/participant/PvModelITSpec.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * © 2020. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.simona.model.participant
+
+import edu.ie3.simona.test.common.UnitSpec
+import org.scalatest.matchers.should.Matchers
+import squants.Each
+import squants.energy.{Megawatts, Power}
+
+/** A simple integration test that uses pre-calculated data to check if the pv
+  * model works as expected. It uses 8 pv models located in GER.
+  */
+
+class PvModelITSpec extends Matchers with UnitSpec with PvModelITHelper {
+
+  private implicit val tolerance: Power = Megawatts(
+    1e-5
+  )
```

Comment:
Be careful here. This would be a tolerance of 10 W, while we had way lower in the groovy test before. Watts(1e-8) should be the right one if I'm not wrong here...